src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml

```diff
@@ -60,6 +61,12 @@ jobs:
           _ExcludeOmniSharpTests: false
           _Platform: linux
           _RunOnline: false
+        CentOSStream9-Offline:
+          _BootstrapPrep: true
+          _BuildArch: x64
+          _Container: ${{ parameters.centOSStream9Container }}
+          _ExcludeOmniSharpTests: false
+          _RunOnline: false
```

Comment:
In [a recent ](https://github.com/dotnet/installer/pull/13153), I added the _Platform parameter.  Can you please include it with this change?  Without it, the msft to sb sdk content comparison tests won't run for this leg.

Reply:
Good catch! Done.